internal/services/mssql/helper/constants.go

```diff
@@ -0,0 +1,19 @@
+package helper
+
+// sku.name is not an Enum in the REST API specs.
+// These are the accepted values based on https://learn.microsoft.com/en-us/azure/azure-sql/database/elastic-jobs-overview?view=azuresql#concurrent-capacity-tiers
+const (
+	SqlJobAgentSkuNameJA100 string = "JA100"
+	SqlJobAgentSkuNameJA200 string = "JA200"
+	SqlJobAgentSkuNameJA400 string = "JA400"
+	SqlJobAgentSkuNameJA800 string = "JA800"
+)
```

Comment:
Are these absent in all API versions?

There are cases where we also open an issue on the Rest API specs to get these added and then reference that issue here, but as usual it's not consistent when we do and don't do that. Suppose the question is whether for something like missing enums it's worth the overhead of opening an issue for tracking - maybe something to discuss in the next team call.